mailpoet/assets/js/src/logs/list.tsx

```diff
@@ -17,7 +17,7 @@ type Log = {
 
 export type Logs = Log[];
 
-function isCtrl(event: MouseEvent): boolean {
+function isCtrl(event: {ctrlKey ?:boolean, metaKey ?:boolean, altKey ?:boolean}): boolean {
```

Comment:
Why `MouseEvent` doesn't work here? Is there a correct type somewhere (like `React.MouseEvent` or something)?

Reply:
The problem for me originated here:
![image](https://user-images.githubusercontent.com/6458412/154007966-08750551-97d9-4a5f-94ec-dc1508bd6f74.png)

I tried to reproduce this in the playground, but I hadn't had any problem there. Not quite sure, what is causing it.